src/blockchain_db/lmdb/db_lmdb.cpp

```diff
@@ -25,6 +25,8 @@
 // STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF
 // THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
+#ifndef APPLE_DEBUG_BLOCKCHAIN_DB
```

Comment:
So this massive `#ifndef` wrap from line `28` to the `#endif` on line `3428` just looks like it prevents the compilation of the whole of db_lmdb.cpp to me. Can you explain why that's not the case?